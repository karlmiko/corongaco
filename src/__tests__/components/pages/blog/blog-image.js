import React from 'react'
import renderer from 'react-test-renderer'
import BlogImage from '~components/pages/blog/blog-image'

describe('Components : Pages : Blog : Blog Image', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <BlogImage
          imageSource="//imagehosting.com/thisimage"
          imageAlt="Test alt text"
          caption="Author Designer / The COVID Tracking Project"
          containerStyle="style-class"
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
