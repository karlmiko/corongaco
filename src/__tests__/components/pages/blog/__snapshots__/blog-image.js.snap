// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : Blog Image renders correctly 1`] = `
<div
  className="style-class"
>
  <img
    alt="Test alt text"
    src="//imagehosting.com/thisimage"
  />
  <div
    className="credit"
  >
    Author Designer / The COVID Tracking Project
  </div>
</div>
`;
